src/applications/check-in/travel-claim/pages/travel-agreement/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+import Wrapper from '../../../components/layout/Wrapper';
+import BackButton from '../../../components/BackButton';
+import { useFormRouting } from '../../../hooks/useFormRouting';
+
+const TravelAgreement = props => {
+  const { router } = props;
+  const { t } = useTranslation();
+
+  const { goToPreviousPage } = useFormRouting(router);
+  const finishReview = () => {
+    goToPreviousPage();
+  };
+
+  return (
+    <>
+      <BackButton
+        router={router}
+        action={goToPreviousPage}
+        prevUrl="#back"
+        text={t('back-to-last-screen')}
+      />
+      <Wrapper
+        pageTitle="Travel Agreement"
+        classNames="travel-page"
+        withBackButton
+      >
+        <ul data-testid="agreement-list-items">
+          <li>{t('i-have-incurred-a-cost-in-relation')}</li>
+          <li>
+            {t('i-have-neither-obtained-transportion-at-government-expense')}
+          </li>
+          <li>{t('i-have-not-received-other-transportion-resources')}</li>
+          <li>{t('i-am-the-only-person-claiming-for-the-travel-listed')}</li>
+          <li>{t('i-have-not-previously-received-payment')}</li>
+        </ul>
+        <a
+          data-testid="file-claim-link"
+          href="#travel-review"
```

Comment:
axe will complain about anchor links without an associated id here. I wonder if we can use react-router-dom's  `Link` here instead. might be worth exploring since we know the url we are going back to. You can see it being used inside of BackButton.

Reply:
I didn't see axe complain about it and didn't see any other areas we're using an `id=` for an anchor tag?